scripts/generate-cmd.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -eux
+docker-entrypoint.sh generate -i /local/transcendence-app/swagger-spec.yaml \
+  -g typescript-fetch \
+  -o /local/webapp/src/shared/generated
+useradd -U -ms /bin/bash "${MY_USER}"
```

Comment:
Maybe this will quit with error if $MY_USER already exists?

Reply:
what do you think about using `$USER` instead? you'll always have the current user assigned to the env

Reply:
Maybe getting the user running in this container and adding a condition to skip the `chown` if the match?
https://stackoverflow.com/questions/14810684/check-whether-a-user-exists

Reply:
Possibly. Will check.